Compiler Bug(?): Static Declaration of 'winmm_reverb_level' Follows Non-Static Declaration

**_Operating System and Version:_** Windows 11 Nightly Build (Build 22621.ni_release220506-1250)

### Bug description
I've been trying my best to follow the wiki's guide on compiling Chocolate Doom (using MSYS32, installing all the dependences [as well as installing a few more dependencies for Crispy, a fork of Chocolate], etc.), but when I try running make, the compiling process always exits with Error 2, the error happening over at the sound.c file.

**_Compiler Log:_**
```
sound.c:85:12: error: static declaration of 'winmm_reverb_level' follows non-static declaration
   85 | static int winmm_reverb_level = 40;
      |            ^~~~~~~~~~~~~~~~~~
In file included from sound.h:18,
                 from sound.c:34:
../../src/i_sound.h:272:12: note: previous declaration of 'winmm_reverb_level' with type 'int'
  272 | extern int winmm_reverb_level;
      |            ^~~~~~~~~~~~~~~~~~
sound.c:86:12: error: static declaration of 'winmm_chorus_level' follows non-static declaration
   86 | static int winmm_chorus_level = 0;
      |            ^~~~~~~~~~~~~~~~~~
../../src/i_sound.h:273:12: note: previous declaration of 'winmm_chorus_level' with type 'int'
  273 | extern int winmm_chorus_level;
      |            ^~~~~~~~~~~~~~~~~~
make[3]: *** [Makefile:437: sound.o] Error 1
make[3]: Leaving directory '/home/dp223/chocolate-doom/src/setup'
make[2]: *** [Makefile:1124: all-recursive] Error 1
make[2]: Leaving directory '/home/dp223/chocolate-doom/src'
make[1]: *** [Makefile:562: all-recursive] Error 1
make[1]: Leaving directory '/home/dp223/chocolate-doom'
make: *** [Makefile:415: all] Error 2
```

--- a/src/i_winmusic.c
+++ b/src/i_winmusic.c
@@ -19,12 +19,16 @@
 
 #include <windows.h>
 #include <mmsystem.h>
+#include <math.h>
 #include <stdio.h>
 #include <stdlib.h>
 
 #include "doomtype.h"
 #include "m_misc.h"
 #include "midifile.h"
+#include "i_sound.h"
+#include "i_winmusic.h"
+
 
 #define BETWEEN(l,u,x) (((l)>(x))?(l):((x)>(u))?(u):(x))
 

--- a/src/i_winmusic.h
+++ b/src/i_winmusic.h
@@ -32,6 +32,6 @@ void I_WIN_UnRegisterSong(void);
 void I_WIN_ShutdownMusic(void);
 
 
-#endif
+#endif // _WIN32
 
-#endif
+#endif // __I_WINMUSIC__

--- a/src/i_winmusic.h
+++ b/src/i_winmusic.h
@@ -31,8 +31,6 @@ boolean I_WIN_RegisterSong(char* filename);
 void I_WIN_UnRegisterSong(void);
 void I_WIN_ShutdownMusic(void);
 
-extern int winmm_reverb_level;
-extern int winmm_chorus_level;
 
 #endif
 

--- a/src/i_winmusic.h
+++ b/src/i_winmusic.h
@@ -14,10 +14,10 @@
 // DESCRIPTION:
 //      Windows native MIDI
 
-#ifdef _WIN32
-
 #ifndef __I_WINMUSIC__
 #define __I_WINMUSIC__
+
+#ifdef _WIN32
 
 #include "doomtype.h"
 

--- a/src/setup/sound.c
+++ b/src/setup/sound.c
@@ -82,8 +82,8 @@ static char **midi_names;
 static int midi_num_devices;
 static int midi_index;
 char *winmm_midi_device = NULL;
-static int winmm_reverb_level = 40;
-static int winmm_chorus_level = 0;
+int winmm_reverb_level = 40;
+int winmm_chorus_level = 0;
 #endif
 
 // DOS specific variables: these are unused but should be maintained

